scripts/bountiful.ash

```diff
@@ -539,6 +540,14 @@ string combat(int round, monster opp, string text) {
   // Check if the current monster is hunted
   if(is_hunted(opp)) {
     print("Hey it's the bounty monster!", "blue");
+
+    // olfact if haven't already for bounty monster. Save 1 olfact for farming purposes
```

Comment:
good shout re: saving an olfact. however, do we want to make the "reserve one for farming" thing a user preference? there are a decent bunch of prefs and this feels preffable.

Reply:
good call, will make it a pref